README.md

```diff
@@ -4,6 +4,14 @@ NYAN TAP output ~~inspired by~~ blatantly copied from Mocha's [nyan reporter](ht
 
 ![screen shot](http://i.imgur.com/T1eH156.png)
 
+**ANSI colors**
```

Comment:
I am not sure wee need to add the `supports-color` flags description. They are not that important imho...